### PR TITLE
feat: render products on Ui from Api

### PR DESCRIPTION
**Pull request by tushartiwari7**
PR for Issue #4 
Created Filters Component for Product Listing Page and replaced rendered data from api with the hardcoded data.
Product-showcase with all components.

localhost_3000_products

### CODE REVIEW

**Review by ArjunGTX — APPROVED** [review]
The code is well structured and clean.

File: src/components/Filters/Filters.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <div className="my-xs  pointer">
+          <input type="radio" id="low" name="sort" value="" />
+          <label htmlFor="low" className="mx-sm fs-s">
+            Low to High
+          </label>
+        </div>
+      </ul>
+      <h5 className="my-md h4">Category</h5>
+      <ul className="list m-md flex flex-col">
+        {categories.map(({ _id, categoryName }) => {
```
Comment (JantuDeb) [inline src/components/Filters/Filters.jsx:32]: This is good that you are not duplicating html.

File: src/services/getCategories.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+export const getCategories = async () => {
+  try {
+    const {data: {categories}} = await axios.get("/api/categories");
+    return categories;
+  } catch (error) {
+    console.log(error);
```
Comment (JantuDeb) [inline src/services/getCategories.js:7]: You may return an object with data or error, so that you can show the error in view.

**Review by JantuDeb — APPROVED** [review]
Overall code looks good, you have separated all the components and the data flow between the components is also managed well.

File: src/services/getCategories.js (hunk 2 above)
Comment (tushartiwari7) [reply src/services/getCategories.js:7]: Sure! its a good idea.

**Pull request merged**